test/e2e/framework/ingress_utils.go

```diff
@@ -1051,10 +1051,34 @@ func (j *IngressTestJig) CreateIngress(manifestPath, ns string, ingAnnotations m
 		j.Ingress.Annotations[k] = v
 	}
 	Logf(fmt.Sprintf("creating" + j.Ingress.Name + " ingress"))
-	j.Ingress, err = j.Client.ExtensionsV1beta1().Ingresses(ns).Create(j.Ingress)
+	j.Ingress, err = j.RunCreate(j.Ingress)
 	ExpectNoError(err)
 }
 
+func (j *IngressTestJig) RunCreate(ing *extensions.Ingress) (*extensions.Ingress, error) {
+	if j.Class != MulticlusterIngressClassValue {
+		return j.Client.ExtensionsV1beta1().Ingresses(ing.Namespace).Create(ing)
+	}
+	// Use kubemci to create a multicluster ingress.
+	// TODO: Store the ingress in a file that can be passed to kubemci.
+	filename := "ing-file"
+	// TODO: Add gcp-project
+	_, err := RunKubemciCmd("create", ing.Name, fmt.Sprintf("--ingress=%s", filename))
+	return ing, err
+}
+
+func (j *IngressTestJig) RunUpdate(ing *extensions.Ingress) (*extensions.Ingress, error) {
```

Comment:
Why do RunCreate and RunUpdate both call "kubemci create"? I think that makes sense, as a kubemci author, but it sure looks confusing as an outside reader.

Reply:
Good point. Added a comment